merlin/systems/dag/ops/faiss.py

```diff
@@ -44,7 +81,38 @@ def from_config(cls, config):
 
         return operator
 
-    def export(self, path, input_schema, output_schema, params=None, node_id=None, version=1):
+    def export(
+        self,
+        path: str,
+        input_schema: Schema,
+        output_schema: Schema,
+        params: dict = None,
+        node_id: int = None,
+        version: int = 1,
+    ) -> List[dict, list]:
+        """
+        Export the class object as a config and all related files to the user defined path.
+
+        Parameters
```

Comment:
ending punctuation, please.

For path, does it need to be a directory?  Does the directory need to exist?

sugg: s/key, value pairs/key and value pairs/  mostly because slash is bad in text and "and" is an OK substitute.